corehq/apps/export/views.py

```diff
@@ -1727,8 +1727,8 @@ def _check_export_size(self, export_instances, filters):
             count += get_export_size(instance, filters)
         if count > MAX_EXPORTABLE_ROWS:
             raise ExportAsyncException(
-                _("This export contains " + count + " rows. Please change the " +
-                "filters to be less than " + MAX_EXPORTABLE_ROWS + "rows.")
+                _("This export contains {count} rows. Please change the filters to be less than {max} "
+                  "rows.".format(count=count, max=MAX_EXPORTABLE_ROWS))
```

Comment:
looks good but you should actually put the format call after the translation, since you want to inject the variables into the translated string.